src/main/java/teammates/ui/controller/AdminSearchPageAction.java

```diff
@@ -266,17 +266,15 @@ private String findAvailableInstructorGoogleIdForCourse(String courseId){
         if(instructorList == null || instructorList.isEmpty()){
             return googleId;
         }
-        
-        for(InstructorAttributes instructor : instructorList){
-          
-            if(instructor.googleId != null){
-               googleId = instructor.googleId;
-               if(instructor.isAllowedForPrivilege(Const.InstructorPermissionRoleNames.INSTRUCTOR_PERMISSION_ROLE_COOWNER)){            
-                   break;
-               } 
-            }            
+
+        for (InstructorAttributes instructor : instructorList) {
+
+            if (instructor.isRegistered() && instructor.isAllowedForPrivilege(Const.ParamsNames.INSTRUCTOR_PERMISSION_MODIFY_INSTRUCTOR)) {
```

Comment:
Yes, but the issue wants the page to use the Co-owner instructor, so we should be testing for that instead right? 

We could compare the privileges to check if the instructor has the same permissions as the Co-owner